snuba/datasets/transactions_processor.py

```diff
@@ -135,6 +135,12 @@ def _process_base_event_values(
         processed["duration"] = max(int(duration_secs * 1000), 0)
 
         processed["platform"] = _unicodify(event_dict["platform"])
+
+        group_ids = event_dict["data"].get("group_ids") or []
```

Comment:
I've noticed there's already `group_id` on transaction and it's not in `data`. Does it make sense to expect `group_ids` to be in `data`?

Reply:
The answer to this question depends on how data is written to the eventstream. I do not have the background on why certain fields were taken out from `data` and put into the outer message. But if on the eventstream you decide to write the `group_ids` only on the outer part of the message, you do not need to find it from `data`